index.ts

```diff
@@ -311,10 +313,62 @@ export const User = {
   },
 };
 
+// Handles case where owner is user and case where owner is org
+const getOwner = (gref) => {
+  const { name: owner } = 'name' in gref.$argsAt(root.users.one) ? gref.$argsAt(root.users.one) : gref.$argsAt(root.organizations.one);
+  return owner;
+}
+
+// const getOwnerGref = (owner) => {
+//   return
+// }
+
+const getRepos = async (obj, args) => {
+  if ('name' in obj.$argsAt(root.users.one)) {
+    const { name: username } = obj.$argsAt(root.users.one);
+
+    const apiArgs = toGithubArgs({ ...args, username });
+    const res = await client().repos.listForUser(apiArgs);
+    return res
+
+  }
+  else {
+    const { name: org } = obj.$argsAt(root.organizations.one);
+
+    const apiArgs = toGithubArgs({ ...args, org });
+    const res = await client().repos.listForOrg(apiArgs);
+    return res
+  }
+}
```

Comment:
This could be a bit cleaner with something like:
```suggestion
const getRepos = async (gref, args) => {
  const owner = getOwner(gref);
  const isUser = "name" in gref.$argsAt(root.users.one);
  if (isUser) {
    const apiArgs = toGithubArgs({ ...args, username: owner });
    return await client().repos.listForUser(apiArgs);
  }

  const apiArgs = toGithubArgs({ ...args, org: owner });
  return await client().repos.listForOrg(apiArgs);
};
```

Reply:
Eh, on second thought it might be more readable the way you have it, maybe just tightened up a bit:
```suggestion
const getRepos = async (obj, args) => {
  if ('name' in obj.$argsAt(root.users.one)) {
    const { name: username } = obj.$argsAt(root.users.one);
    const apiArgs = toGithubArgs({ ...args, username });
    return await client().repos.listForUser(apiArgs);
  } else {
    const { name: org } = obj.$argsAt(root.organizations.one);
    const apiArgs = toGithubArgs({ ...args, org });
    return await client().repos.listForOrg(apiArgs);
  }
}
```